### PR TITLE
simple storybook instaruction and few missing libraries

### LINKED ISSUE #3: storybook have some missing dependency
I had to add this 3 library to run storybook

packages/react-components/package.json
```json
    "@emotion/styled": "11.9.3",
    "@emotion/react": "11.9.3",
    "framer-motion": "6.5.1",
```

if you want I will send PR

### PR DESCRIPTION
**Pull request by matart15**
close #3

### CODE REVIEW

**Comment by matart15** [comment]
I saw `packages/*/package-lock.json` files were added, So I added `package-lock.json` on top too. let me know if you want me to remove it

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
 # nstal
+
```
Comment (matart15) [inline README.md:2]: very simple instruction about how to start with stroybook

File: package.json [hunk 2]
```diff
@@ -3,5 +3,8 @@
   "private": true,
   "devDependencies": {
     "lerna": "^4.0.0"
+  },
+  "dependencies": {
+    "typescript": "4.7.4"
```
Comment (matart15) [inline package.json:8]: `npx lerna bootstrap` required it

**Comment by phbernard** [comment]
Thank you Matar for this PR!

**Pull request merged**